plugins/out_azure_kusto/azure_kusto_conf.c

```diff
@@ -199,10 +199,13 @@ static int parse_storage_resources(struct flb_azure_kusto *ctx, struct flb_confi
     }
 
     jsmn_init(&parser);
-    tokens = flb_calloc(1, sizeof(jsmntok_t) * tok_size);
+    //tokens = flb_calloc(1, sizeof(jsmntok_t) * tok_size);
+
+    /* Dynamically allocate memory for tokens based on response length */
+    tokens = flb_calloc(1, sizeof(jsmntok_t) * (flb_sds_len(response)));
 
     if (tokens) {
-        ret = jsmn_parse(&parser, response, flb_sds_len(response), tokens, tok_size);
+        ret = jsmn_parse(&parser, response, flb_sds_len(response), tokens, flb_sds_len(response));
```

Comment:
How about storing a length of string in tok_size variable?
Counting the length in every time is inefficient to process.